.github/workflows/benchmark.yml

```diff
@@ -28,7 +28,8 @@ jobs:
                   env = Dict(
                       "JULIA_NUM_THREADS" => "2",
                   ),
-              )
+              ),
+              "origin/tk/optimal-filter",
```

Comment:
This needs to be reverted before merging, I used it to benchmark against the optimal filter branch